Ошибка: HTTPConnectionPool(host='g', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x00000161C4FFFFD0>: Failed to resolve 'g' ([Errno 11001] getaddrinfo failed)"))
URL: http://g